app/mailers/admins/account_mailer.rb

```diff
@@ -1,9 +0,0 @@
-class Admins::AccountMailer < Mail::Notify::Mailer
-  def account_creation_request(record, _opts = {})
-    @template = NOTIFY_ADMIN_ACCOUNT_CREATION_REQUEST_TEMPLATE
```

Comment:
Please delete this constant from initializers/notification.rb too, and, once merged, you could delete the template from Notify itself.